homeassistant/components/darksky/sensor.py

```diff
@@ -402,6 +419,81 @@ def get_state(self, data):
         return state
 
 
+class DarkSkyAlertSensor(Entity):
```

Comment:
Would this fit better as a binary sensor? Alert / no alert.

Reply:
True point.  It does make sense as a binary sensor.  Let me give it a try.

Reply:
I think it works better as a true sensor as it's possible your location can have multiple alerts/weather advisories at once.  Here we've been in a flash flood watch and a thunderstorm warning at the same time, giving two alerts.